src/openrct2/management/Research.cpp

```diff
@@ -817,6 +817,32 @@ bool scenery_is_invented(uint16 sceneryItem)
     return invented;
 }
 
+bool scenery_set_is_invented(sint32 scenerySetIndex)
```

Comment:
With "scenery_set" do you mean scenery group (like "Abstract theming")? If so, please call it scenery group in order to reduce ambiguity - set in programming context is usually interpreted as a verb, not a noun.

Reply:
done